index.js

```diff
@@ -36,5 +36,7 @@ module.exports = function (hex, options = {}) {
 
 	return options.format === 'array' ?
 		[red, green, blue, alpha] :
-		{red, green, blue, alpha};
+		options.format === 'text' ?
+			`rgb(${red}, ${green}, ${blue}${alpha === 255 || alpha === 1 ? '' : `, ${alpha}`})` :
```

Comment:
You can remove `alpha === 255 || `. See: https://github.com/sindresorhus/hex-rgb/commit/068420a3680a7d00e4b24704f8c32f43e4b00d87

Reply:
`rgb(` => `rgba(` to support alpha and to always be consistent.